05LessonFive/Checkpoint1/index.html

```diff
@@ -5,6 +5,14 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div class="box" id="lbox">
+    <img src="img/berries.jpg"/>
+  </div>
 
+  <div class="box" id="rbox">
+    <h1 id="berries">Berries</h1>
```

Comment:
great initial effort here, Harry! I would want you to be more efficient with your code, however. because you already have unique ids and classes for your containers, consider going through those so you don't have to id and/or class every element. for example #rbox h1  or #rbox > p